inc/classes/class-google-auth.php

```diff
@@ -158,6 +158,17 @@ protected function _get_user_from_token( $token ) {
 				'picture'      => $google_userinfo->getPicture(),
 			];
 
+			/**
+			 * This hook provides access token fetched by google sign-in.
+			 *
+			 * @since 0.1
```

Comment:
Do we need here version `0.2`?

Reply:
No it'll be 1.0, since 1.0 will be the new version we'll be releasing

Reply:
I'll be raising version update PR shortly